src/content/docs/zh-cn/guides/imports.mdx

```diff
@@ -142,7 +206,7 @@ Astro 支持使用浏览器的 [`WebAssembly`](https://developer.mozilla.org/en-
 
 ## Node 内置模块
 
-我们鼓励 Astro 用户尽可能避免使用 Node.js 内置模块（`fs`、`path` 等.）。Astro 计划在未来与多个 JavaScript 运行时兼容。包括 [Deno](https://deno.land/) 和 [Cloudflare Workers](https://workers.cloudflare.com/)，它们不支持诸如 `fs` 等 Node 内置模块。
+我们鼓励 Astro 用户尽可能避免使用 Node.js 内置模块（`fs`、`path` 等）。Astro 兼容多个运行时使用 [适配器](/zh-cn/guides/server-side-rendering)。这包括 [Deno](/zh-cn/guides/integrations-guide/deno/) 和 [Cloudflare Workers](/zh-cn/guides/integrations-guide/cloudflare/)，它们不支持 Node 内置模块，例如 `fs`。
```

Comment:
This will update via a new PR after https://github.com/withastro/docs/pull/2863 merged.

```
/zh-cn/guides/server-side-rendering#添加一个适配器
```